### ISSUE TITLE
v5: In a fresh install, trying to view a PDF generates a PDF of the login page

### ISSUE THREAD

**Issue by hmnd** [invoiceninja/dockerfiles#208]
For some reason, upon freshly installing Invoice Ninja 5.0.24 from Docker, generating a PDF of an invoice results in a PDF of the login page. I'm not sure where to start with debugging this or where I might find logs, so feel free to let me know what I can provide.

**Comment by turbo124** [comment]
@hmnd 

We are updating the dockerfile currently sorting out a few of these issues.

The first thing you may want to try is the following:

```
 sudo docker-compose exec app php artisan storage:link
```

**Comment by hmnd** [comment]
@turbo124 I've tried both that and `php artisan optimize`, but neither has helped unfortunately.

**Comment by turbo124** [comment]
@hmnd are you able to paste your docker-compose.yml (please redact any sensitive data)

**Comment by hmnd** [comment]
@turbo124 I'm not actually using docker-compose. I've got a separate container for Caddy with the configuration below, and another container with `linuxserver/mariadb`. It's also worth noting that I did have PDFs working in the 5.0.17 version of the image.

```
(billing_invoiceninja) {
  root * /var/www/app/public
  file_server browse

  php_fastcgi http://billing-invoiceninja:8000

  encode zstd gzip
}
```

**Comment by turbo124** [comment]
@hmnd I'm not sure what to suggest if you are not using the provided compose file, perhaps inspect the configs in this repo and fix any of the differences.

**Comment by hmnd** [comment]
@turbo124 So this is really weird... I added a log to `app/Jobs/Entity/CreateEntityPdf.php`, hoping that I could figure something out that way. I copied my modified file into the container, and that magically fixed it. I then removed the container and recreated and it's still working, even though I don't have `app` persisted in a volume. No clue what I did, but I guess it works now! Thanks for your help haha.
